### PR TITLE
Add tutorial 1-7: Coordinate Systems

### PR DESCRIPTION
**Pull request by Vassalware**
This PR adds the OpenTK port of the LearnOpenGL Coordinate Systems tutorial.

### CODE REVIEW

**Comment by devvoid** [comment]
Looks fantastic! No complaints from me.

File: learn/chapter1/7-coordinate-systems.md [hunk 1]
```diff
@@ -0,0 +1,292 @@
...
+
+![Perspective Projection](img/7-perspective.png)
+
+As you can see, due to perspective the lines seem to coincide the farther they're away. This is exactly the effect perspective projection tries to mimic and it does so using a perspective projection matrix. The projection matrix maps a given frustum range to clip space, but also manipulates the w value of each vertex coordinate in such a way that the further away a vertex coordinate is from the viewer, the higher this w component becomes. Once the coordinates are transformed to clip space they are in the range -w to w (anything outside this range is clipped). OpenGL requires that the visible coordinates fall between the range -1.0 and 1.0 as the final vertex shader output, thus once the coordinates are in clip space, perspective division is applied to the clip space coordinates:
+
+![Perspective Division](img/7-perspective_division.png)
```
Comment (Perksey) [inline learn/chapter1/7-coordinate-systems.md]: I'm getting a 404 on this image.

File: learn/chapter1/7-coordinate-systems.md [hunk 2]
```diff
@@ -0,0 +1,292 @@
...
+
+- To move a camera backwards, is the same as moving the entire scene forward.
+
+That is exactly what a view matrix does, we move the entire scene around inversed to where we want the camera to move.
+Because we want to move backwards and since OpenGL is a right-handed system we have to move in the positive z-axis. We do this by translating the scene towards the negative z-axis. This gives the impression that we are moving backwards.
+
+> **Right-handed system**
+> By convention, OpenGL is a right-handed system. What this basically says is that the positive x-axis is to your right, the positive y-axis is up and the positive z-axis is backwards. Think of your screen being the center of the 3 axes and the positive z-axis going through your screen towards you. The axes are drawn as follows:
+> ![Right-Handed Coordinate System](img/7-coordinate_systems_right_handed.png)
+>To understand why it's called right-handed do the following:
```
Comment (Perksey) [inline learn/chapter1/7-coordinate-systems.md]: Needs another blank line to ensure that it's not on the same line as the image.
image

**Comment by Vassalware** [comment]
Fixed those issues. Looks like the vscode markdown preview extension I was using isn't as accurate as the stock one; I wasn't getting the text on the same line as the image until I switched back.

And don't misunderstand: I didn't write the article. It's almost identical to the source content at http://learnopengl.com, just edited slightly to use C# and OpenTK.

**Pull request merged**